mobile/android/app/src/main/kotlin/com/example/mobile/BackgroundServicePlugin.kt

```diff
@@ -44,30 +39,30 @@ class BackgroundServicePlugin : FlutterPlugin, MethodChannel.MethodCallHandler {
     override fun onMethodCall(call: MethodCall, result: MethodChannel.Result) {
         val ctx = context!!
         when(call.method) {
-            "initialize" -> { // needs to be called prior to any other method
+            "enable" -> {
                 val args = call.arguments<ArrayList<*>>()!!
                 ctx.getSharedPreferences(BackupWorker.SHARED_PREF_NAME, Context.MODE_PRIVATE)
-                    .edit().putLong(BackupWorker.SHARED_PREF_CALLBACK_KEY, args.get(0) as Long).apply()
+                    .edit()
```

Comment:
The question specifically for Kotlin; is the way you are calling the method a consider a chaining method call? Can you do

```Kotlin
val a = ctx.getSharedPreferences().edit()
val b = a.putLong()
```

Reply:
yes, it's method chaining similar to the typical builder pattern